ui/admin/app/components/chat/Chatbar.tsx

```diff
@@ -69,7 +69,7 @@ export function Chatbar({ className }: ChatbarProps) {
 						}
 					}}
 					maxHeight={200}
-					minHeight={0}
+					minHeight={32}
```

Comment:
can you use `rows` instead so we get the natural row height?

```suggestion
					rows={1}
```